docker/srv/salt/install-packages.sls

```diff
@@ -1,4 +1,9 @@
+{%- if grains['os_family'] == 'Debian' %}
+  {%- set package = "vim" %}
+{%- elif grains['os_family'] == 'RedHat' %}
+  {%- set package = "vim-enhanced" %}
+{%- endif %}
```

Comment:
There must be a default, which is `vim`. But in that case the statement can be even slightly simplified.

Reply:
`vim` is a virtual package and looks it's not working properly with CentOS (you can check by yourself). What would be your proposition according simplification? It's quite simple and follow https://docs.saltstack.com/en/latest/topics/best_practices.html (except exposing package name with context).